subsys/bluetooth/services/fast_pair/fp_crypto/include/fp_crypto.h

```diff
@@ -140,11 +142,13 @@ size_t fp_crypto_account_key_filter_size(size_t n);
  *			       equal to number of Account Keys.
  * @param[in] n Number of account keys (n >= 1).
  * @param[in] salt Random byte - Salt.
+ * @param[in] battery_info Battery info or NULL if there is no battery info. Length of battery info
+ *			   must be equal to @ref FP_CRYPTO_BATTERY_INFO_LEN.
  *
  * @return 0 If the operation was successful. Otherwise, a (negative) error code is returned.
  */
 int fp_crypto_account_key_filter(uint8_t *out, const struct fp_account_key *account_key_list,
-				 size_t n, uint8_t salt);
+				 size_t n, uint8_t salt, const uint8_t *battery_info);
```

Comment:
What about using packed structure instead of buffer like:
```
struct battery info {
    uint8_t info;
   uint8_t values[3];
} __packed;

```suggestion
				 size_t n, uint8_t salt, const struct *battery_info);
```

This is just a suggestion.

Reply:
I think that the structure would eventually be added, but it could be internal to fp_advertising. The meaning of info (data len and type) and battery level values fields is not relevant to fp_crypto. The crypto module treats it as 4 bytes which are input for crypto operations.

Reply:
Can we leave it as it is?

Reply:
We can